WcaOnRails/db/migrate/20180709220826_add_created_at_column_to_competitions.rb

```diff
@@ -0,0 +1,5 @@
+class AddCreatedAtColumnToCompetitions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :Competitions, :created_at, :datetime
```

Comment:
I think we could use `add_timestamps` here instead: https://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/add_timestamps. This would create both a `created_at`, and an `updated_at`. Currently we don't have any plans for the updated_at, but it's kind of a Rails-y thing do have both of them. The only reason the Competitions table doesn't have both of them is because it existed before we switched to Ruby on Rails.